sympy/solvers/tests/test_solveset.py

```diff
@@ -1636,3 +1636,8 @@ def test__is_finite_with_finite_vars():
 
 def test_issue_13550():
     assert solveset(x**2 - 2*x - 15, symbol = x, domain = Interval(-oo, 0)) == FiniteSet(-3)
+
+
+def test_issue_13842():
```

Comment:
This issue does not seem to involve `nonlinsolve`. Is the issue number correct?

Reply:
Yes, I have changed the typing error. @jksuom